lib/node_modules/@stdlib/math/base/special/cflipsign/manifest.json

```diff
@@ -24,17 +26,56 @@
 	],
 	"confs": [
 		{
-			"src": [
-				"./src/main.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [],
-			"libpath": [],
-			"dependencies": [
-        "@stdlib/math/base/assert/is-negative-zero"
-      ]
+		  "task": "build",
+		  "src": [
+			"./src/main.c"
+		  ],
+		  "include": [
```

Comment:
Mixed TABS and spaces. Ensure EditorConfig is properly setup.

Reply:
Update: mixed tabs and spaces seems to stem from the `manifest.json` file using old conventions and then the more recent EditorConfig conventions overriding, leading to mixed behavior. We should do a sweep across the project to normalize indentation in these files to prevent further copy-paste errors.

In the meantime, if you can convert indentation to spaces, that would be appreciated!

Reply:
I'll open an issue for this one!